### PR TITLE
Find best loan rates

### PR DESCRIPTION
**Pull request by AndyGabler**
Added a page to view all of the offers of loans, otherwise referred to as loan templates.

### CODE REVIEW

**Review by frap129 — APPROVED** [review]
Looks good to me, but I'm not super familiar with PHP so I'd wait for someone else to approve before merging

**Review by genocidalist — APPROVED** [review]
Looks fine here

**Pull request merged**
